website/cue/reference/configuration.cue

```diff
@@ -251,16 +251,28 @@ configuration: {
 				}
 			}
 		}
-		log_namespacing: {
-			common:      false
+		schema: {
+			common: false
 			description: """
-				Globally enables / disables log namespacing. See [Log Namespacing](\(urls.log_namespacing_blog))
-				for more details. If you want to enable individual sources, there is a config
-				option in the source configuration.
+				Configures options for how Vector handles schema.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ 
```suggestion
				Configures options for how Vector handles event schemas.
```

Reply:
[dc63455](https://github.com/vectordotdev/vector/pull/17961/commits/dc63455feaf8881145cb0ce78cfa9a09b76cbb35)